lib/active_model/serializable/utils.rb

```diff
@@ -4,13 +4,15 @@ module Utils
       extend self
 
       def _const_get(const)
-        begin
-          method = RUBY_VERSION >= '2.0' ? :const_get : :qualified_const_get
-          Object.send method, const
-        rescue NameError
-          const.safe_constantize
+        Serializer.serializers_cache.fetch_or_store(const) do
```

Comment:
There are two places where _const_get is called, so [ActiveModel::Serializable#namespace](https://github.com/rails-api/active_model_serializers/blob/c2565ed72623eb4ecc9d21149778aa160a9ad9b2/lib/active_model/serializable.rb#L36) and [ActiveModel::Serializer.serializer_for](https://github.com/rails-api/active_model_serializers/blob/c2565ed72623eb4ecc9d21149778aa160a9ad9b2/lib/active_model/serializer.rb#L68).  That is why I decided to move it into _const_get instead of doing it in ActiveModel::Serializer.serializer_for.  Should I be duplicating the code between both call sites?  Or only worry about the latter? Since I am not sure if the former is actually a performance problem.